docs/decisions/0006-ledger-balance-transaction-states.rst

```diff
@@ -0,0 +1,37 @@
+0006 Ledger balance and transaction states
+##########################################
+
+Status
+******
+
+**Accepted** (May 2023)
+
+Context
+*******
+
+We will take a strong position on the states of transactions that should count when
+computing the balance of a ledger.
+
+Decision
+********
+
+Those states will be all non-failed states:
```

Comment:
nit: bit difficult to understand what this means at first. The sentence is referring to the sentence in the "Context" section above, but isn't immediately obvious imo. Perhaps be a bit more explicit in this description?